src/structure.rs

```diff
@@ -43,39 +68,45 @@ impl StructureOptions {
     ) -> Self {
         let mut bitfield = 0 as OptionsBitfield;
         if include_hetatm {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_INCLUDE_HETATM; 
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_INCLUDE_HETATM;
         }
         if include_hydrogen {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_INCLUDE_HYDROGEN;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_INCLUDE_HYDROGEN;
         }
         if separate_models {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_SEPARATE_MODELS;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_SEPARATE_MODELS;
         }
         if separate_chains {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_SEPARATE_CHAINS;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_SEPARATE_CHAINS;
         }
         if join_models {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_JOIN_MODELS;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_JOIN_MODELS;
         }
         if halt_at_unknown {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_HALT_AT_UNKNOWN;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_HALT_AT_UNKNOWN;
         }
         if skip_unknown {
-            bitfield = bitfield | freesasa_structure_options_FREESASA_SKIP_UNKNOWN;
+            bitfield = bitfield
+                | freesasa_structure_options_FREESASA_SKIP_UNKNOWN;
         }
         if radius_from_occupancy {
             bitfield = bitfield | freesasa_structure_options_FREESASA_RADIUS_FROM_OCCUPANCY;
         }
-        Self{
-            bitfield
-        }
+        Self { bitfield }
     }
 }
 
 impl Default for StructureOptions {
+    /// Defaults StructureOptions bitfield to 0
```

Comment:
In the future you don't need to bother with comments for these standard library trait implementations. 

You are welcome to if you want, but it isn't super necessary

Reply:
That's good to know!